### ISSUE TITLE
[Directory History]: paths from WFX plugins aren't being saved

### ISSUE THREAD

**Issue by PetbkA** [doublecmd/doublecmd#1194]
### Version

Double Commander
Version: **1.1.0 beta**
Revision: 32
Commit: **cd3a69c65**
Build date: 2023/08/05
Lazarus: 2.2.7.0
FPC: 3.2.2
Platform: **x86_64-Win64-win32/win64**
OS version: **Windows 10 Home 22H2 x86_64**

### Description

Paths from WFX plugins aren't being saved in _Directory History_ (_cm\_DirHistory_). This would be fine for e.g. [Services2](https://totalcmd.net/plugring/Services2.html) plugin, but for FTP / [Secure FTP](https://totalcmd.net/plugring/sftp4tc.html) inability to see dirs history is sometimes inconvenient

**Comment by alexx2000** [comment]
Yes, `cm_DirHistory` works with local file system only. While tab specific `cm_ViewHistory` works with all file systems, but it has its own characteristics.
